scripts/pylib/twister/twisterlib/hardwaremap.py

```diff
@@ -287,11 +287,11 @@ def readlink(link):
         serial_devices = list_ports.comports()
         logger.info("Scanning connected hardware...")
         for d in serial_devices:
-            if d.manufacturer in self.manufacturer:
+            if d.manufacturer.upper() in [m.upper() for m in self.manufacturer]:
```

Comment:
you probably want to use casefold() for comparison.

Reply:
Ok, I will change it to casefold().

Reply:
done